compiler/one-cmds/utils.py

```diff
@@ -184,10 +186,10 @@ def _make_circle2circle_cmd(args, driver_path, input_path, output_path):
             if type(getattr(args, opt[0])) is bool:
                 cmd.append('--' + opt[0])
             """
-            This condition check is for config file interface, usually would be 
-             SomeOption=True 
-            but user can write as follows while development 
-             SomeOption=False 
+            This condition check is for config file interface, usually would be
+             SomeOption=True
+            but user can write as follows while development
+             SomeOption=False
```

Comment:
does this change come from format checker?

Reply:
No. I ran `./nnas format` with previous PRs but these trailing spaces were not removed. 😸 

I created another PR to remove these trailing spaces. #7294